### ISSUE TITLE
Cannot Reproduce quality of pretrained model

### ISSUE THREAD

**Issue by chazo1994** [KevinMIN95/StyleSpeech#18]
I have trained a stylespeech model use LibriTTS, but the quality was far worse than pretrain stylespeech model of author. I use default config and parameter and train the model within 100k step. The loss like bellow:
![image](https://user-images.githubusercontent.com/17020334/158014894-58426534-2a79-4bce-9842-d4c38db68fd7.png)
![image](https://user-images.githubusercontent.com/17020334/158014904-9f28bc46-8dea-45e4-ad0d-fd75cbe90a3f.png)

I also upload my audio sample of the text same as demo page in folder Train_LibriTTS_StyleSpeech in attached fille. There are always strange sounds at the end of each audio file, i can't explain that.
[meta_stylespeech_results.zip](https://github.com/KevinMIN95/StyleSpeech/files/8237476/meta_stylespeech_results.zip)

**Comment by KevinMIN95** [comment]
I think you should train the model on more steps. In my experiments, the mel loss decreased to 0.22~0.23.

**Comment by chazo1994** [comment]
> I think you should train the model on more steps. In my experiments, the mel loss decreased to 0.22~0.23.

Thanks, I will continue training to 400k step and show the results. But I still confused why I use same dataset, default parameter and train to 100k step like step in pretrained model but loss and quality is worse.

**Comment by chazo1994** [comment]
I Found that, the problem come from Montreal Forced Aligner version 2.0.0a22 or newer version, which do not put “sp” or “sil” in the phone tier. To fix this, just add "--disable_textgrid_cleanup" flag during alignment step.

**Comment by KevinMIN95** [comment]
Oh. Thanks for finding the problem! I should update the repo to match with libraries of the newer versions.

**Comment by sirius0503** [comment]
@chazo1994 : Even if ['sp', 'sil'] are put in the **phones** tier, they are anyways ignored in the `get_alignment` function in `preprocessors/utils.py` - so if it is there or not shouldn't matter?
